core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -2539,7 +2539,7 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
           ArgumentMatchers.eq(SamResourceTypeNames.billingProject),
           ArgumentMatchers.eq(billingProjectName),
           ArgumentMatchers.eq(SamBillingProjectActions.createWorkspace),
-          ArgumentMatchers.argThat(userInfoEq(testContext))
+          any[RawlsRequestContext]
```

Comment:
Why did you make this change?

Reply:
because the context passed to this function is not the same as the test context. I'm less keen on prescribing request contexts as it makes refactoring less easy.